docs/user/config-file-glossary.rst

```diff
@@ -107,6 +107,10 @@ Message settings
 ``IRC_SHOW_ZWSP=true``
     Prevents users with the same Telegram and IRC username from pinging themselves across platforms.
 
+``IRC_SHOW_LOCATION_MESSAGE=false``
+    If a user shares their location on Telegram, this will forward the GPS coordinates
+    of their location to IRC if set to true.
```

Comment:
Are we still following the [one-sentence-per-line convention](https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line)? I can't remember if that is documented. If so, 111-112 be changed to one line for an easier `git diff` if/when the docs are updated in the future.

```suggestion
If a user shares their location on Telegram, this will forward the GPS coordinates of their location to IRC if set to true.
```